pkg/scheduler/service/service.go

```diff
@@ -547,7 +547,7 @@ func (s *SchedulerServer) bootstrap(hks hyperkube.Interface) (*ha.SchedulerProce
 		ReconcileCooldown: s.ReconcileCooldown,
 	})
 
-	masterUri := kmcloud.MasterURI()
+	masterUri := kmcloud.CloudProvider.MasterURI()
```

Comment:
now that we're using the cloud provider to get this URI, we probably need to make sure it's initialized before we attempt to access it.

https://github.com/GoogleCloudPlatform/kubernetes/blob/release-0.14/pkg/cloudprovider/plugins.go#L66